test/gtest/device_api/single_write_test.cu

```diff
@@ -195,20 +201,48 @@ protected:
     }
 
     void
-    completeWireup(size_t from_agent, size_t to_agent) {
-        nixl_notifs_t notifs;
-        nixl_status_t status = getAgent(from_agent).genNotif(getAgentName(to_agent), NOTIF_MSG);
-        ASSERT_EQ(status, NIXL_SUCCESS) << "Failed to complete wireup";
-
-        do {
-            nixl_status_t ret = getAgent(to_agent).getNotifs(notifs);
-            ASSERT_EQ(ret, NIXL_SUCCESS) << "Failed to get notifications during wireup";
-            std::this_thread::sleep_for(std::chrono::milliseconds(10));
-        } while (notifs.size() == 0);
+    completeWireup(size_t from_agent, size_t to_agent,
+                  const std::vector<MemBuffer> &wireup_src,
+                  const std::vector<MemBuffer> &wireup_dst) {
+        nixl_opt_args_t warmup_params = {};
+
+        for (size_t worker_id = 0; worker_id < num_workers_; worker_id++) {
+            warmup_params.customParam = "worker_id=" + std::to_string(worker_id);
+
+            nixlXferReqH *warmup_req = nullptr;
```

Comment:
It seems that the initialization is not needed, since there is `ASSERT_EQ(status, NIXL_SUCCESS)`.